### DISCUSSION TITLE
Ability to shift lane left or right to line up with edge of field?

### DISCUSSION THREAD

**Discussion by frazierzak** [Mogli12/KeyboardSteer#757] [Q&A] [answered]
Would it be possible to hit a key combination like SHIFT+A or SHIFT+D to nudge the working line left or right a meter at a time?

**Comment by Mogli12** [comment, marked as answer]
Did you try KEY_rshift KEY_left and KEY_rshift KEY_right?

**Comment by frazierzak** [comment]
I apologize. That's what I get for not reading the in-game help section you so kindly created 👍 

Thanks
